.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I suggest putting this in your local git exclude?

Reply:
The idea was to help somebody who will do the same using vscode and [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker), but I removed that 👌